src/main/resources/view/AddRecipeForm.fxml

```diff
@@ -12,51 +11,58 @@
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
-
-<AnchorPane xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
-    <ScrollPane styleClass="root-layout" hbarPolicy="NEVER"
-                AnchorPane.topAnchor="0.0" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0">
-        <VBox xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" styleClass="root-box"
-            spacing="8"
-            >
+<AnchorPane
+        xmlns:fx="http://javafx.com/fxml/1" maxWidth="Infinity"
+        maxHeight="Infinity" xmlns="http://javafx.com/javafx/8">
+    <stylesheets>
+        <URL value="@Form.css"/>
+    </stylesheets>
+    <ScrollPane
+            styleClass="root-layout" hbarPolicy="NEVER"
+            fitToWidth="true" fitToHeight="true"
+            AnchorPane.bottomAnchor="-1.0" AnchorPane.topAnchor="0.0"
+            AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0">
+        <VBox xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/8" styleClass="root-box"
+              spacing="8"
```

Comment:
xmlns redundant here since anchorpane already has it

Reply:
fixed, removed